app/src/main/java/org/thoughtcrime/securesms/conversationlist/ConversationListFragment.java

```diff
@@ -226,7 +226,9 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
     initializeTypingObserver();
     initializeSearchListener();
 
-    RegistrationLockDialog.showReminderIfNecessary(this);
+    if (!TextSecurePreferences.isPinV2ReminderDisabled(requireContext())) {
+      RegistrationLockDialog.showReminderIfNecessary(this);
+    }
```

Comment:
Better to do this check inside the body of the method `showReminderIfNecessary`

Reply:
That is a static method so I'd have to pass the context there. It can be done but will require more maintenance in the future.

Reply:
OK, if you like. I just copied the method OWS uses in onResume() for the sms reminder.